numba/tests/test_sets.py

```diff
@@ -666,6 +653,14 @@ def _range(self, stop):
         return list(zip(b, c, d))
 
 
+class TestUnicodeSets(OtherTypesTest, BaseTest):
```

Comment:
I would like to see a couple additional tests to check all the cases where incref/decrefs are being inserted. Can you verify which methods are tested and add additional tests as need?